src/resolvers-cassandra/Settings/index.js

```diff
@@ -27,5 +29,6 @@ module.exports = {
   trustedTwitterAccounts: queries.trustedTwitterAccounts,
   facebookPages: queries.facebookPages,
```

Comment:
`facebookPages` is a list of trusted sources so not sure why this is still needed.

Reply:
Just to confirm, `facebookPages` is not needed? I understand that now we have a `trustedSources` query so should that mean that `facebookPages` and `trustedTwitterAccounts` should be removed, since they are covered by the `trustedSources` query?